lib/tasks/ea_lookups.rake

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+namespace :ea_lookups do
```

Comment:
@Cruikshanks I am not super sure this is the correct name for the scope, I wanted to keep consistent with WEX, but there we will have a task for `easting and northing` which I understand won't be added here?

Reply:
Agree, great idea to go with a namespace here that is consistent with WEX.

I'd suggest dropping the `ea_` though because in WEX the x & y task will be calling OS Places and not anything to do with the Environment Agency.

So my suggestions would be

- `lookups`
- `area_lookup`
- `missing`
- `location`

> _Naming things is hard!_

Reply:
Great suggestions! Thanks a ton!